vllm/model_executor/models/ultravox.py

```diff
@@ -363,6 +379,9 @@ def __init__(self, *, vllm_config: VllmConfig, prefix: str = ""):
             hf_config=config.text_config,
             prefix=maybe_prefix(prefix, "language_model"),
         )
+        # logits_processor is added here to support 'lm_head' LoRA module
+        # for language model
+        self.logits_processor = self.language_model.logits_processor
```

Comment:
@jeejeelee i had to add this because vllm fails [here](https://github.com/vllm-project/vllm/blob/main/vllm/lora/models.py#L493). Do you know why lm_head is special? Can you think of any negative impact with this change on ultravox? 

cc @Yard1 which has a comment to make this more robust in code that linked above